Mathlib/RingTheory/Ideal/QuotientOperations.lean

```diff
@@ -932,3 +932,47 @@ theorem quotQuotEquivQuotOfLE_symm_comp_mkₐ (h : I ≤ J) :
 
 end AlgebraQuotient
 end DoubleQuot
+
+namespace Ideal
+
+section PowQuot
+
+variable {R : Type*} [CommRing R] (I : Ideal R) (n : ℕ)
+
+/-- I ^ n ⧸ I ^ (n + 1) is equivalent as a quotient of I ^ n or as an ideal of
+R ⧸ I ^ (n + 1). -/
```

Comment:
I had a bit of trouble parsing the docstring upon first reading. Not sure if the following is an improvement though
```suggestion
/-- `I ^ n ⧸ I ^ (n + 1)` can be viewed as a quotient module and as ideal of `R ⧸ I ^ (n + 1)`.
This definition gives the `R`-linear equivalence between the two. -/
```